### DISCUSSION TITLE
server: Profile Subscription Feature (asciicast post notifications)

### DISCUSSION THREAD

**Discussion by BinaryBrewWorks** [asciinema/discussions#204] [Ideas]
Good day,

A profile subscription feature would be a great way to notify 'followers' as to when new asciicasts have been posted.  More than just an RSS feed, it could encourage user signups as a prerequisite to subscribing to profiles (with notifications going to their registered email).

Or an ATOM or RSS feed would do as well =)  Cheers! 

<img width="428" alt="screen shot 2015-12-19 at 11 09 56 pm" src="https://cloud.githubusercontent.com/assets/5445806/11916458/aa2d7d22-a6a5-11e5-8e36-52fb040b719a.png">

**Comment by ku1ik** [comment]
Atom feed seems to be a quick win so I'll prioritize this. Profile subscriptions inside the app is also great idea!
